sparse_ho/criterion/held_out.py

```diff
@@ -54,6 +78,29 @@ def get_val(self, model, X, y, log_alpha, monitor=None, tol=1e-3):
     def get_val_grad(
             self, model, X, y, log_alpha, get_beta_jac_v, max_iter=10000,
             tol=1e-5, compute_jac=True, monitor=None):
+        """Get value and gradient of criterion.
+
+        Parameters
+        ----------
+        model: instance of ``sparse_ho.base.BaseModel``
+            A model that follows the sparse_ho API.
+        X: np.array-like, shape (n_samples, n_features)
+            Design matrix.
+        y: np.array, shape (n_samples,)
+            Observation vector.
+        log_alpha: float or np.array
+            Logarithm of hyperparameter.
+        get_beta_jac_v: function
+            Returns the product of the transpoe of the Jacobian and a vector v.
+        max_iter: int
+            Maximum number of iteration for the inner problem.
+        tol: float, optional (default=1e-3)
+            Tolerance for the inner problem.
+        compute_jac: bool
+            To compute or not the Jacobian.  # This should be removed
```

Comment:
in front of "this should be removed"
I just noticed that there was a todo in another occurrence so that's OK

Reply:
ha ok! I will out another TODO to be sure then